INSTALL.md

```diff
@@ -1,3 +1,19 @@
+# Obtaining google-cloud-cpp-spanner
+
+There are two primary ways of obtaining `google-cloud-cpp-spanner`. You can use git:
+
+```bash
+git clone git@github.com:googleapis/google-cloud-cpp-spanner.git $HOME/project
+```
+
+Or obtain the tarball release:
+
+```bash
+mkdir -p $HOME/project
+wget -q https://github.com/googleapis/google-cloud-cpp-spanner/archive/v0.4.0.tar.gz
+tar -xf v0.4.0.tar.gz -C $HOME/project
```

Comment:
I think this will result in a new subdirectory under `$HOME/project` named like `google-cloud-cpp-spanner-v0.4.0/` that then contains the files.

We could fix this using `tar -xf v0.4.0.tar.gz -C $HOME/project --strip 1`, which *I think* works on all the Linux flavors?

Or we could just extract into $HOME, then move the named dir to "project".

Reply:
Thanks. FYI, that makes you the [second Google employee](https://meta.stackexchange.com/questions/214729/hello-from-google-spam-or-no) to call me Remy. Feel free to call me Tom.

Reply:
Oops, my apologies. Not sure why I thought that, but thanks for correcting me, Tom.